src/utils/create-utils.ts

```diff
@@ -399,6 +400,16 @@ export const createImageRepository = (
           ? ImageRepositoryVisibility.private
           : ImageRepositoryVisibility.public,
       },
+      notifications: [
```

Comment:
@sahil143 does the notification configuration is hardcoded?
If so, this is not something we can have for the community version.